.github/actions/unit-tests/action.yaml

```diff
@@ -18,12 +18,30 @@ runs:
         pip install -r requirements/dev.txt
       shell: bash
 
-    - name: Run Tox lint commands
+    - name: Run linting
       # Run tox using the version of Python in `PATH`
       run: tox -e lint
       shell: bash
 
-    - name: Run Pytest through Tox
+    - name: Should run doc build
+      run: |
+        import os
+        from packaging.version import Version
+        with open(os.getenv("GITHUB_ENV"), "a") as file_env:
```

Comment:
This needed since one of the plugins for doc building requires python 3.10 or greater